src/module/proficiencyUtils.ts

```diff
@@ -30,17 +30,6 @@ export function prepareSkillOptions(optionSettings: OptionSettings) {
   );
 }
 
-export function prepareToolOptions(optionSettings: OptionSettings) {
-  const foundryTools = (game as any).dnd5e.config.toolProficiencies;
-  const toolChoices: KeyValue[] = Object.keys(foundryTools).map((k) => ({ key: k, value: foundryTools[k] }));
-  prepareOptions(
-    optionSettings,
-    'toolProf',
-    optionSettings.filteredOptions ?? toolChoices,
-    game.i18n.localize('DND5E.TraitToolProf'),
-  );
-}
-
```

Comment:
moved this to have all proficiencies with types together (they will later be all async to get some data from the items) so tools/weapons/armor moved together with skills and languages apart.